--- anotherone.py	2020-03-11 10:18:14.932523896 -0500
+++ anotherone2.py	2020-03-11 13:04:24.356280185 -0500
@@ -8,5 +8,4 @@
 
 print(check_zip_code("The zip codes for New York are 10001 thru 11104.")) # True
 print(check_zip_code("90210 is a TV show")) # False
-print(check_zip_code("Their address is: 123 Main Street, Anytown, AZ 85258-0001.")) # True
-print(check_zip_code("The Parliament of Canada is at 111 Wellington St, Ottawa, ON K1A0A9.")) # False
\ No newline at end of file
+print(check_zip_code("Their address is: 123 Main Street, Anytown, AZ 85258-0001.")) # True
\ No newline at end of file
